src/utility.ts

```diff
@@ -38,6 +39,22 @@ export class Utility {
 
 }
 
+export class EventCounter {
+    public static dict: {[key: string]: number} = {};
+
+    public static async increase(event: string) {
+        try {
+            await this._lock.acquire();
```

Comment:
why do we need the lock here?

Reply:
Now it works fine without lock, I add it just in case of asynchronous operations in future. For example, if we want to reuse this class, and add `decrease` method to this class, a lock would make it safer.

Reply:
If there is no async invocations, there is no need to do that, given that js is single thread. You can check about the `execution stack, event loop` in js for more details.